src/Markdig/Markdig.targets

```diff
@@ -6,7 +6,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>0.18.3</VersionPrefix>
     <Authors>Alexandre Mutel</Authors>
-    <TargetFrameworks>net35;net40;netstandard2.0;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
Out of curiosity, what .NET Core 2.1 specific APIs are you using in the lib?
Wouldn't .NET Standard 2.0 be enough to replace that target too?

And as for .NET Core 3.1, I assume that's because you have some vectorized paths using SSE/AVX/etc.?

Reply:
Some APIs like `TextWriter.Write(Span)` aren't in standard 2.0 so that's why I added core2.1 before.
Core 3.1 is used only for string.GetPinnableReference in CharacterMap rn

Reply:
Right now there's no manual hw intrinsics code in Markdig, but that can change now that we're refreshing build targets

Reply:
I see, targeting .NET Core 2.1 directly as a workaround to be able to use those `TextWriter.Write(Span<T>)` overloads makes sense, yeah. As for .NET Core 3.1 though, if it's just for that single `string.GetPinnableReference` call, wouldn't it make more sense to drop that target, just target .NET Standard 2.1, and pass `ReadOnlySpan<char>` values around (instead of `string`-s), and then just use `MemoryMarshal.GetReference(ReadOnlySpan<T>)` to get a mutable `ref` from there? This would also allow you to just split/tokenize over `ReadOnlySpan<T>` instead of allocating substrings, in case you're doing that too.

Just a thought! 😄

Reply:
StringSlice is the type plummed throughout Markdig to avoid substring allocations. It provides good utility like the NextChar method that you would otherwise have to replace with slices/more manual index bookkeeping.

I'm not too worried about having the extra target since I can see us adding more code relying on new APIs in the future

Reply:
I see, yeah that makes perfect sense. I was just curious, thank you for taking the time to provide more info, I appreciate it! 😊